src/hooks/useAuth.tsx

```diff
@@ -41,16 +37,10 @@ export function AuthProvider({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [isLoading]);
 
-    // TODO
-    // useEffect(() => {
-    //     // get current user with api call
-    //     const getUser = async () => {
-    //         const user = await getCurrentUser();
-    //         setUser(user);
-    //     };
-
-    //     getUser();
-    // }, []);
+    useEffect(() => {
+        const user = getUserClaims();
```

Comment:
The useEffect doesnt seem to be happening

Reply:
I'm still getting every user attribute as undefined

Reply:
dang. okay setting on login now, should be fixed